### DISCUSSION TITLE
ANNOUNCEMENT TIME

### DISCUSSION THREAD

**Discussion by JWSwan** [JWSwan/TheNeighborhood#30] [Show and tell]
Hear ye, Hear ye:

Hey everyone, due to the intricacies hitherto unknown of our most gracious host - Heroku, @danielewartwelch needed to create a separate repository for the TTCB website. Furthermore, we found out that in order to push updates to Heroku, the Neighborhood needed to be an 'Organization.' So...congratulations! The Neighborhood is now one step closer to being a fully independent commune. 

Basically, we'll still be using this repository to brainstorm/store/work, but in order for things to be made 'real,' they'll need to end up in the Neighborhood 'Organization.' You should all be getting invites to the Neighborhood Organization in your Github-associated-email soon. I imagine this was not explained well and you'll have questions, place them below pls thx.
